src/equiv.js

```diff
@@ -12,8 +12,11 @@ QUnit.equiv = (function() {
 	function useStrictEquality( b, a ) {
 
 		/*jshint eqeqeq:false */
-		if ( b instanceof a.constructor || a instanceof b.constructor ) {
+		if ( b instanceof a.constructor && a instanceof b.constructor ) {
 
+			// Compare same constructor.
+			return a.toString() == b.toString();
```

Comment:
@gibson042 
Thank you. I think of it again. I think this don't have to check type. Could you please explain to me more detail about check type? And How about think below logic?
```js
function useStrictEquality( b, a ) {
	if ( b instanceof a.constructor || a instanceof b.constructor ) {
		return a.valueOf() === b.valueOf();
	} else {
		return a === b;
	}
}
```